auth-web/src/store/modules/business.ts

```diff
@@ -178,11 +178,11 @@ export default class BusinessModule extends VuexModule {
 
       for (const affiliationInviteInfo of pendingAffiliationInvitations) {
         const isFromOrg = affiliationInviteInfo.fromOrg.id === this.currentOrganization.id
-        const isToOrgAndPending = affiliationInviteInfo.toOrg.id === this.currentOrganization.id &&
+        const isToOrgAndPending = affiliationInviteInfo?.toOrg?.id === this.currentOrganization.id &&
```

Comment:
This code needs to be refactored for code complexity, managed to do that in my latest PR